### PR TITLE
Schema Diagram pull request

### PR DESCRIPTION
**Pull request by AleWaweru**
- Created a schema diagram using inbuild PostgreSQL tools
- Created a file called schema_based_on_diagram.sql to store data used to create the diagram

Screenshot (49)

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  CONSTRAINT fk_treatment_id FOREIGN KEY (treatment_id) REFERENCES treatments (id)
+);
+
+CREATE TABLE treatments (
+  id SERIAL PRIMARY KEY,
+  type varchar(50) NOT NULL,
+  name varchar(50) NOT NULL
+);
+
+
```
Comment (IngDavidTM) [inline schema_based_on_diagram.sql]: - [ ] While you did a fantastic job :+1:, as a project requirement all "join tables" for the many-to-many relationship should be created, even if they are not shown on the diagram. There fore you should create a table between medical_histories and treatments  (name can be different):

```sql
  CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );
```

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi Team 🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitFlow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (AleWaweru) [reply schema_based_on_diagram.sql]: done.

Screenshot (51)

**Review by gemmen29 — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**